packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -218,6 +218,12 @@ export interface GridSimpleOptions {
    * @default "client"
    */
   sortingMode: GridFeatureMode;
+  /**
+   * If positive, the Grid will throttle updates coming from `apiRef.current.updateRows` and `apiRef.current.setRows`.
+   * It can be useful if you have a high update rate but do not want to do heavy work like filtering / sorting or rendering on each  individual update.
```

Comment:
```suggestion
   * It can be useful if you have a high update rate but do not want to do heavy work like filtering/sorting or rendering on each individual update.

```